### PR TITLE
Added the solution to Inorder traversal in C

### PR DESCRIPTION
**Pull request by cherryWood55**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description
I have added the code for Inorder traversal in C - both Recursive and Iterative implementation.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#415

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@cherryWood55 add time and space complexity for both techniques. I think you have implemented recursive and non-recursive both inorder traversal so avoid using names `inorder.c` and `inorder2.c` rather you can use `inorder_recursive.` and `inorder_non_recursive.c`.

**Comment by cherryWood55** [comment]
@iamrajiv I have incorporated the changes into my code, please review.

**Comment by atarax665** [comment]
Update README.md.

**Comment by cherryWood55** [comment]
@atarax665 I have previously created a PR for updation of README. Please look into #621 .

**Comment by Kajol-Kumari** [comment]
> @atarax665 I have previously created a PR for updation of README. Please look into #621 .

Hey @cherryWood55 You need to add you code's path in the readme. @atarax665 meant to say this. So, please update the readme for this PR

**Review by Kajol-Kumari — COMMENTED** [review]
please add one sample input-output below the complexity section for each of these.

**Comment by cherryWood55** [comment]
I have added one Sample input and output @Kajol-Kumari . Please review.

**Comment by cherryWood55** [comment]
Please review this PR @maintainers.

File: inorder_recursive.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	printf ("%d, ", head->info);
+
+	inorder_traversal(head->right);
+
+}
+
+//driver program
+int main(){
+	
+	struct node* head = create_new_node(40);
```
Comment (Kajol-Kumari) [inline inorder_recursive.c:38]: Can we please take these input from user?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@cherryWood55 Please add your code path in `readme.md` for reference you can take a look at any other PR.

**Pull request closed without merging**